code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/EdgePersonalizationResponseHandler.java

```diff
@@ -56,16 +57,25 @@ class EdgePersonalizationResponseHandler {
     private final MessagingCacheUtilities messagingCacheUtilities;
     private final ExtensionApi extensionApi;
     private final LaunchRulesEngine launchRulesEngine;
-    private final FeedRulesEngine feedRulesEngine;
+    private final ContentCardRulesEngine contentCardRulesEngine;
 
     private Map<Surface, List<Proposition>> propositions = new HashMap<>();
```

Comment:
nit: We can rename this to `cachePropositions` or `inAppAndCBEPropositionsBySurface` to ensure we don't accidentally omit adding it to result of `retrieveMessages`